### PR TITLE
Added Terminal Commands Cheat Sheet

### PR DESCRIPTION
**Pull request by midhruvjaink**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [ ] New Instant Answer
  - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
  - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
  - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**
###### Description of changes

What changes does this PR introduce? 
###### Related Issues or Discussions

What issues or discussions (if any) does this PR address? Please provide the issue # or link.
###### People to notify

@mention any relevant people/organizations

---

Instant Answer Page: https://duck.co/ia/view/{{ID}}

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
@midhruvjaink Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/terminal.json [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        "Installing from Sources",
+        "Stopping and Starting"
+    ],
+    "sections":{
+        "System Info":[{
+            "val":"Show current date and time",
+            "key":"[date]"
+        },{
+            "val":"Show this month's calendar",
+            "key":"[cal]"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/terminal.json]: These aren't keys, are you sure you're using the right template?

**Comment by GuiltyDolphin** [comment]
@midhruvjaink This doesn't look like a terminal cheat sheet - it includes various commands which are independent of the terminal.

You've mentioned a couple of package managers too, but these have nothing to do with the terminal, and aren't very generic.

We already have the `linux` cheat sheet, which covers various misc. Linux commands, and a couple of cheat sheets for specific package managers, perhaps you could help improve those?

**Comment by pjhampton** [comment]
Closing as this PR looks to have gone stale.

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**